web/modules/custom/ct_github/src/Plugin/ContributionSource/GithubContribution.php

```diff
@@ -74,14 +85,18 @@ public static function create(ContainerInterface $container, array $configuratio
       $plugin_id,
       $plugin_definition,
       $container->get('entity_type.manager'),
-      $container->get('ct_github.query')
+      $container->get('ct_github.query'),
+      $container->get('config.factory')
     );
   }
 
   /**
    * Get all users with github username.
    */
   public function getUsers() {
+    if (!$this->tokenValid) {
+      return [];
+    }
```

Comment:
This is good because if we don't return any users, no items are queued and this plugin is not called again. Can you add a comment clarifying this?
```suggestion
    // Return nothing because we can't do anything with any users if we
    // don't have a valid token.
    if (!$this->tokenValid) {
      return [];
    }
```